app/dao/notifications_dao.py

```diff
@@ -136,6 +135,28 @@ def dao_get_7_day_agg_notification_statistics_for_service(service_id,
 
 
 @statsd(namespace="dao")
+def dao_get_template_usage(service_id, limit_days=None):
```

Comment:
a brief check suggests that limit_days is either not passed in or set to 7 - ideally i'd say replace with a boolean flag, then we can just either select from `Notification` or `NotificationHistory` - saves having to filter on `created_at`